smartdashboard/views.py

```diff
@@ -159,7 +160,7 @@ def status(self) -> str:
         """
         return get_experiment_status_summary(self.runs)
 
-    def update(self) -> None:
+    def update_status(self) -> None:
```

Comment:
I think `update_status` might be better as an internal/private method based on the way the subclases implement it and the base class executes it during `update`. Should it be named `_update_status` to clarify that private nature?

Reply:
...based on the `LiveView` comment above... `EntityView` could forego having the `@abc.abstractmethod:def update_status` and only LiveView implementations would have it, reducing the need to implement a no-op.

Reply:
Updated the name to `_update_status`!